### ISSUE TITLE
How to implement?

### ISSUE THREAD

**Issue by CamGenius** [OpenUAS/castlelinklive4arduino#1]
Been trying to get this to work for a few days with a Mamba X and Sidewinder 4 but have been unable to connect to the esc. I have enabled link live in the castle link software and am running with an Arduino Uno. I have the signal pin of the esc plugged into pin 12 as that is what the example seems to show for the throttle input pin.

I just keep getting this no matter what I do.

```
Waiting for data... No data
Waited (ms): 500
```

Any help would be greatly appreciated, thanks!

**Comment by RamBotTech** [comment]
> Been trying to get this to work for a few days with a Mamba X and Sidewinder 4 but have been unable to connect to the esc. I have enabled link live in the castle link software and am running with an Arduino Uno. I have the signal pin of the esc plugged into pin 12 as that is what the example seems to show for the throttle input pin.
> 
> I just keep getting this no matter what I do.
> 
> ```
> Waiting for data... No data
> Waited (ms): 500
> ```
> 
> Any help would be greatly appreciated, thanks!

Hey Cam, Ive been trying the same for the newer ESCS, mamba x8s, mamba monster x8s, mamba x, and cannot get this working. Were you able to make progress? I saw you had also made your own project for telemetry data from these ESC's, did this project work for there newer escs aswell?

**Comment by CamGenius** [comment]
Yes mine is pretty rudimentary but it did work with some esp32's
